blueprints/model-test/index.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Blueprint for generating a test for a frost ember-data adapter
```

Comment:
is this for a model-test or adapter?

I see it listed as an adapter test in the comment above and the `description` below.